train_graph.py

```diff
@@ -237,19 +237,25 @@ def train(params, args, local_rank, world_rank, world_size):
 if __name__ == '__main__':
 
   parser = argparse.ArgumentParser()
-  parser.add_argument("--run_num", default='00', type=str)
-  parser.add_argument("--yaml_config", default='./config/UNet.yaml', type=str)
-  parser.add_argument("--config", default='base', type=str)
-  parser.add_argument("--no_val", action='store_true', help='skip validation steps (for profiling train only)')
+  parser.add_argument("--run_num", default='00', type=str, help='tag for indexing the current experiment')
+  parser.add_argument("--yaml_config", default='./config/UNet.yaml', type=str, help='path to yaml file containing training configs')
+  parser.add_argument("--config", default='base', type=str, help='name of desired config in yaml file')
   parser.add_argument("--enable_amp", action='store_true', help='enable automatic mixed precision')
   parser.add_argument("--enable_apex", action='store_true', help='enable apex fused Adam optimizer')
   parser.add_argument("--enable_jit", action='store_true', help='enable JIT compilation')
   parser.add_argument("--enable_benchy", action='store_true', help='enable benchy tool usage')
+  parser.add_argument("--enable_manual_profiling", action='store_true', help='enable manual nvtx ranges and profiler start/stop calls')
   parser.add_argument("--data_loader_config", default=None, type=str,
                       choices=['synthetic', 'inmem', 'lowmem', 'dali-lowmem'],
                       help="dataloader configuration. choices: 'synthetic', 'inmem', 'lowmem', 'dali-lowmem'")
+  parser.add_argument("--local_batch_size", default=None, type=int, help='local batchsize (manually override global_batch_size config setting)')
+  parser.add_argument("--num_epochs", default=None, type=int, help='number of epochs to run')
+  parser.add_argument("--num_data_workers", default=None, type=int, help='number of data workers for data loader')
   args = parser.parse_args()
-  
+
+  if (args.enable_benchy and args.enable_manual_profiling):
+      raise "Enable either benchy profiling or manual profiling, not both."
```

Comment:
You can't raise a string. Can you just wrap that string in a `RuntimeError(...)`?

Reply:
That is my code. I can fix it.

Reply:
we'll fix in a follow-up